koku/masu/test/processor/aws/test_aws_report_db_cleaner.py

```diff
@@ -60,150 +59,154 @@ def test_initializer(self):
-            first_bill = self.accessor._get_db_obj_query(bill_table_name).first()
-            cutoff_date = first_bill.billing_period_start
-
-            self.assertIsNotNone(self.accessor._get_db_obj_query(bill_table_name).first())
-            self.assertIsNotNone(self.accessor._get_db_obj_query(line_item_table_name).first())
-            self.assertIsNotNone(self.accessor._get_db_obj_query(cost_entry_table_name).first())
-
-        removed_data = cleaner.purge_expired_report_data(cutoff_date, simulate=True)
-
-        self.assertEqual(len(removed_data), 1)
-        self.assertEqual(removed_data[0].get("account_payer_id"), first_bill.payer_account_id)
-        self.assertEqual(removed_data[0].get("billing_period_start"), str(first_bill.billing_period_start))
-
-        with schema_context(self.schema):
-            self.assertIsNotNone(self.accessor._get_db_obj_query(bill_table_name).first())
-            self.assertIsNotNone(self.accessor._get_db_obj_query(line_item_table_name).first())
-            self.assertIsNotNone(self.accessor._get_db_obj_query(cost_entry_table_name).first())
-
-    def test_purge_expired_report_data_for_provider(self):
-        """Test that the provider_uuid deletes all data for the provider."""
-        bill_table_name = AWS_CUR_TABLE_MAP["bill"]
-        line_item_table_name = AWS_CUR_TABLE_MAP["line_item"]
-        cost_entry_table_name = AWS_CUR_TABLE_MAP["cost_entry"]
-
-        cleaner = AWSReportDBCleaner(self.schema)
-
-        with schema_context(self.schema):
-            # Verify that data is cleared for a cutoff date == billing_period_start
-            first_bill = self.accessor._get_db_obj_query(bill_table_name).order_by("-billing_period_start").first()
-
-            self.assertIsNotNone(self.accessor._get_db_obj_query(bill_table_name).first())
-            self.assertIsNotNone(self.accessor._get_db_obj_query(line_item_table_name).first())
-            self.assertIsNotNone(self.accessor._get_db_obj_query(cost_entry_table_name).first())
-
-            expected_count = (
-                self.accessor._get_db_obj_query(bill_table_name).filter(provider_id=self.aws_provider_uuid).count()
-            )
-
-        removed_data = cleaner.purge_expired_report_data(provider_uuid=self.aws_provider_uuid)
-
-        self.assertEqual(len(removed_data), expected_count)
-        self.assertIn(first_bill.payer_account_id, [entry.get("account_payer_id") for entry in removed_data])
-        self.assertIn(
-            str(first_bill.billing_period_start), [entry.get("billing_period_start") for entry in removed_data]
-        )
-
-        with schema_context(self.schema):
-            self.assertIsNone(self.accessor._get_db_obj_query(bill_table_name).first())
-            self.assertIsNone(self.accessor._get_db_obj_query(cost_entry_table_name).first())
-            self.assertIsNotNone(self.accessor._get_db_obj_query(line_item_table_name).first())
+    # TODO: Cody, double check to see if we need these
```

Comment:
All of the TODOs in this file need to be addressed.

Reply:
These actually need to be deleted because we won't have a line item table anymore.